### DISCUSSION TITLE
Climatology data on AWS

### DISCUSSION THREAD

**Discussion by sherrieF** [dtcenter/METplus#1323] [File I/O] [answered]
Good Evening,

     No problems just a quick question.  :-)
     Is the climatology data also on the AWS some where.
On Hera it is in /scratch1/NCEPDEV/global/glopara/fix/fix_verif/climo_files/cmean_1d.1959{valid?fmt=%m%d}.

Thanks,
Sherrie

**Comment by j-opatz** [comment]
Hi Sherrie,

And thank you for your question. Could you provide a little more description about which use case this is in regards to? The directory you pointed to on Hera isn't in the METplus input data directory, so I wasn't quite sure what files you're trying to find.

**Comment by sherrieF** [comment]
Good afternoon,

    I was just wondering if there is any climatology data available on the
AWS for doing anomaly correlations.

Thanks,
Sherrie

**Comment by j-opatz** [comment, marked as answer]
Hi Sherrie,

Since we work on building and development of the METplus suite, we don't support/control any of the data not directly used for the support of the METplus suite. The only data we make available to users are the data needed to successfully run the use cases and unit tests for the system.

However, one of the team members did manage to find this AWS [registry](https://registry.opendata.aws/collab/noaa/) of NOAA-provided data, which does have some climatology data available for use. If you need more information regarding the content of those data, or where you may be able to find a specific climatology, I'd suggest reaching out to one of the scientists tied to the center publishing the data.

**Comment by sherrieF** [comment]
I understand.  Thanks very much fo the AWS NOAA registry,
Cheers,
Sherrie
